src/shared-components/container/style.ts

```diff
@@ -24,10 +24,6 @@ const messageStyle = css`
 
 export type ContainerType = 'clickable' | 'message';
 
-/**
- * TODO: Determine if we should return a style by default
- * instead of no style at all.
- */
 const getContainerTypeStyles = (type?: ContainerType) => {
```

Comment:
After looking at the story it is clear we support containers without a type, even if it isn't standard in our usage.